src/plugins/intel_cpu/thirdparty/llmdnn/src/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cmake_minimum_required(VERSION 3.13)
+project(llmdnn)
+
+file(GLOB_RECURSE SOURCE_FILES ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
+
+add_library(llmdnn STATIC ${SOURCE_FILES})
+target_include_directories(llmdnn PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}
+                                  INTERFACE ${CMAKE_CURRENT_SOURCE_DIR}/../include)
```

Comment:
such interface headers will break static build.
We need to wrap it with BUILD_INTERFACE generator expressions.

Reply:
Done.